src/app/fyle/personal-cards/personal-cards.page.ts

```
@@ -241,7 +239,10 @@ export class PersonalCardsPage implements OnInit, AfterViewInit, OnDestroy {
         } else {
           queryParams = params.queryParams as Record<string, string>;
         }
-        queryParams = this.apiV2Service.extendQueryParamsForTextSearch(queryParams as {}, params.searchString);
+        queryParams = this.extendQueryParamsService.extendQueryParamsForTextSearch(
+          queryParams as {},
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)